### PR TITLE
Add hint about replacing example.com

### PR DESCRIPTION
**Pull request by maximiliankolb**
Fixes #3187

@Imaanpreet Does that make sense to you? Do you want me to backport this?

#### What changes are you introducing?

Refs https://github.com/theforeman/foreman-documentation/pull/3187

#### Checklists

* [ ] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [ ] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/common/modules/proc_configuring-pcp-data-collection.adoc [hunk 1]
```diff
@@ -61,6 +61,8 @@ endif::[]
 # echo "https://_{foreman-example-com}_/metrics" > config.d/foreman.url
 # ./Install
 ----
++
+Replace _{foreman-example-com}_ with the FQDN of your {ProjectServer}.
```
Comment (Lennonka) [inline guides/common/modules/proc_configuring-pcp-data-collection.adoc:65]: Why not to use the snippet?

Comment (Imaanpreet) [reply guides/common/modules/proc_configuring-pcp-data-collection.adoc:65]: seems good to me.

Comment (maximiliankolb) [reply guides/common/modules/proc_configuring-pcp-data-collection.adoc]: Added the snippet.

**Review by Lennonka — APPROVED** [review]
LGTM

**Pull request merged**